packages/compass-aggregations/package.json

```diff
@@ -7,28 +7,27 @@
   "main": "lib/index.js",
   "exports": {
     "webpack": "./src/index.js",
+    "browser": "./lib/browser.js",
     "require": "./lib/index.js"
   },
   "scripts": {
-    "clean": "rimraf lib",
-    "precompile": "npm run clean",
-    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js",
-    "prestart": "electron-rebuild --force --only keytar",
-    "start": "cross-env NODE_ENV=development webpack-dev-server --config ./config/webpack.dev.config.js",
-    "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
-    "test:dev": "cross-env NODE_ENV=test mocha-webpack",
-    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
-    "test:karma": "cross-env NODE_ENV=test karma start",
-    "cover": "nyc npm run test",
-    "check": "npm run lint && npm run depcheck",
-    "preanalyze": "mkdir -p .ghpages && cross-env NODE_ENV=production webpack --profile --json --config ./config/webpack.prod.config.js > .ghpages/stats.json",
-    "analyze": "webpack-bundle-analyzer .ghpages/stats.json --no-open  --report .ghpages/report.html --mode static --bundleDir lib",
     "prepublishOnly": "npm run compile",
-    "lint": "eslint \"./src/**/*.{js,jsx}\" \"./test/**/*.js\" \"./electron/**/*.js\" \"./config/**/*.{js,jsx}\"",
+    "compile": "npm run webpack -- --mode production",
+    "prewebpack": "rimraf ./lib",
+    "webpack": "webpack-compass",
+    "start": "npm run webpack serve -- --mode development",
+    "analyze": "npm run webpack -- --mode production --analyze",
+    "typecheck": "tsc --noEmit",
     "depcheck": "depcheck",
-    "test-ci": "npm run test",
-    "bootstrap": "npm run compile"
+    "eslint": "eslint",
+    "lint": "npm run eslint .",
+    "check": "npm run typecheck && npm run lint && npm run depcheck",
```

Comment:
You might notice that I didn't add prettier here, my thinking was to avoid making a giant diff here. If y'all think it would be good to have it here right away too and run it for the plugin I can do this in a separate PR